src/test/regress/expected/multi_modifying_xacts.out

```diff
@@ -119,8 +119,7 @@ EXCEPTION
         RAISE NOTICE 'caught not_null_violation';
 END $$;
 NOTICE:  caught not_null_violation
-COMMIT;
-ERROR:  cannot ROLLBACK TO SAVEPOINT in transactions which modify distributed tables
+ROLLBACK;
 \set VERBOSITY default
```

Comment:
@jasonmp85 Do you recall what you were trying to test? It's a bit weird that we're catching & ignoring an error here...